addons/sale/views/sale_portal_templates.xml

```diff
@@ -201,7 +201,7 @@
 
                     <!-- modal relative to the actions sign and pay -->
                     <div role="dialog" class="modal fade" id="modalaccept">
-                        <div class="modal-dialog" t-if="sale_order.has_to_be_signed(True)">
+                        <div class="modal-dialog modal-md" t-if="sale_order.has_to_be_signed(True)">
```

Comment:
That class does not exist

Reply:
It's not the only place it's used, but I'll remove it from here.